### DISCUSSION TITLE
Pass MIDI from one arduino to another

### DISCUSSION THREAD

**Discussion by Titouan-dev** [FortySevenEffects/arduino_midi_library#320] [Q&A]
Hi,
I'm trying to connect to arduino with midi (I have built an synthetizer with mozzi and a sequencer and want to connect each other).
I have tried multiple thing but I can't get something to work.
My question is do I need an input Midi setup (with an octocoupler) ?
Thanks !

**Comment by franky47** [comment]
Assuming their Serial levels are compatible (eg: both 5V, or both 3.3V, or RX accepting whatever level TX is sending), then yes, I've had success in connecting them directly.

MIDI is both a logical protocol (which sequence of bytes means what) and a physical transport layer specification (DIN plugs, current loop, octocouplers), but for a two-Arduino connection you don't need to bother with the physical layer part of the spec, it's all just a matter of connecting one's TX to the other's RX.
